### PR TITLE
[INTEL MKL] AVX512 Fix for Eigen Vectorized sqrt/rsqrt Function

### PR DESCRIPTION
**Pull request by mdfaijul**
This PR has two modifications:
1. Downloads latest Eigen library that has AVX512 fix for vectorized sqrt/rqrt functions.

2. Added a new unit-test python script [tensorflow/python/kernel_tests/sgd_optimizers_test.py] that tests several SGD optimizers calling sqrt/rsqrt function. The test fails with older Eigen library [https://bitbucket.org/eigen/eigen/get/6913f0cf7d06.tar.gz], but passes with new downloads.

3. In order to check, use the following commands
`bazel test --config=mkl --copt="-mfma" --copt="-march=native" -c opt //tensorflow/python/kernel_tests:sgd_optimizers_test
`
`bazel test --copt="-mfma" --copt="-march=native" -c opt //tensorflow/python/kernel_tests:sgd_optimizers_test`

### CODE REVIEW

**Comment by mdfaijul** [comment]
@benoitsteiner Could you please check this PR?

**Review by benoitsteiner — DISMISSED** [review]

**Comment by mdfaijul** [comment]
@jhseu I see there is one failure: keras integration test. Will look into this.

**Comment by mdfaijul** [comment]
@jhseu Could you please suggest me how should I address pylint errors.

**Comment by jhseu** [comment]
Can you run pylint locally with this pylintrc file?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/pylintrc

**Comment by mdfaijul** [comment]
@jhseu Have corrected style errors.

**Comment by mdfaijul** [comment]
@benoitsteiner @jhseu Could you please review the code. Looks like all tests passes now.

**Comment by mdfaijul** [comment]
@benoitsteiner @jhseu Any update on review?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner: It has been 14 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner: It has been 29 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner: It has been 44 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 61 days with no activity and the `awaiting review` label has been applied.

**Comment by mdfaijul** [comment]
@jhseu Any progress on this PR?

**Comment by jhseu** [comment]
@benoitsteiner Mind taking another look?

**Comment by mdfaijul** [comment]
@benoitsteiner Any update on this PR.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 15 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 30 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 45 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 60 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 75 days with no activity and the `awaiting review` label has been applied.

**Comment by mdfaijul** [comment]
@jhseu @benoitsteiner Any update on this PR?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner, @rmlarsen: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner, @rmlarsen: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner, @rmlarsen: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner, @rmlarsen: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner, @rmlarsen: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner, @rmlarsen: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @benoitsteiner, @rmlarsen: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 59 days with no activity and the `awaiting review` label has been applied.

**Comment by mdfaijul** [comment]
@jhseu @benoitsteiner Should I close this PR? I think the issue has been fixed by downloading the correct version. However, do you think the test is need to be included?

**Comment by jhseu** [comment]
I think we can close it for now. Up to @rmlarsen if we should still add the test.
